src/geo/ui/widgets/histogram/model.js

```diff
@@ -42,14 +42,10 @@ module.exports = WidgetModel.extend({
   },
 
   _onChangeBinds: function() {
-    this.bind('change:url', function(){
-      if (this.get('sync')) {
-        this._fetch();
-      }
-    }, this);
-    this.bind('change:boundingBox', function() {
-      if (this.get('bbox')) {
-        this._fetch();
+    WidgetModel.prototype._onChangeBinds.call(this);
+    this.bind('change:histogramSizes', function(mdl, isSizesApplied, d) {
```

Comment:
Just double checking here... Is this required? Is anybody listening to the `histogramSizes` that this callback is triggering? The `d` variable in the callback doesn't seem to be used.